backend/layers/common/validation.py

```diff
@@ -22,7 +22,11 @@ def _verify_collection_metadata_fields(
 
     def check(key):
         value = getattr(metadata, key)
-        if check_existence and not value:
+        if check_existence and value is None:
```

Comment:
I am not a fan of this method, but this should check two things:
1. The field must exist if `check_existence` is true (namely, if this is a collection _creation_ where all the fields are required)
2. If a field exists, it cannot be falsey (namely, an empty string).

Maybe it would be better to split this behavior into two separate methods, rather than two if conditions.